.github/workflows/main.yml

```diff
@@ -29,60 +31,58 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-14]
+        os: [ubuntu-latest, windows-latest, macos-latest]
     steps:
-      - name: Set up repository
-        uses: tree-sitter/parser-setup-action@v1.2
-        with:
-          node-version: 20
+      - name: Clone repository
+        uses: actions/checkout@v4
       - name: Clone nvim help files
         uses: actions/checkout@v4
         with:
           repository: neovim/neovim
           path: examples/neovim
           sparse-checkout: runtime/doc/
-      - name: Run tests
+      - name: Set up tree-sitter
+        uses: tree-sitter/setup-action/cli@v1
+      - name: Run parser tests
         uses: tree-sitter/parser-test-action@v2
         with:
-          test-library: ${{runner.os == 'Linux'}}
-          corpus-files: |-
+          test-rust: ${{runner.os == 'Linux'}}
+      - name: Parse sample files
+        uses: tree-sitter/parse-action@v4
+        if: runner.os != 'Windows'
```

Comment:
I think this or something equivalent was here before the PR, but this seems weird to me. If some of the files can't correctly be parsed on windows (but can on linux), then to me that still indicates that there is a problem. Right now we cannot prevent regressions on windows since we don't test it.

Reply:
I could make it `continue-on-error` on Windows so you can still check the summary for failures.

Reply:
That still won't solve the problem of regressions, and would also condition people into accepting CI failures so I don't think that's optimal. I still don't understand the benefit of ignoring the failures on windows if I'm being honest.